File: exercicios_py/secao3/exercicio6.py
#Algoritmo que pergunte quando você ganha por hora, horas trabalhadas,
#calcule e mostre o total do seu salário no referido mês.

ht = int(input("Digite quantas horas você trabalhou: "))
gh = int(input("Digite quantos reais você ganha por hora: "))
print("Seu salário nesse mês foi de %.2f reais"%(ht*gh))

File: exercicios_c/secao6/exercicio6.c
//Elabore um algoritmo que leia c e n, código e número de horas trabalhadas. 
//Calcule o salário sabendo-se que ele ganha RS 10,00 por hora. Quando exceder 50, calcule
//o excesso na variável e. Caso contrário, zerar. A hora excedente vale 20. Imprimir
//salário total e excedente.

#include <stdio.h>
int main(){
    int ht, e = 0, st = (ht-e)*10;

    printf("Digite o numero de horas trabalhadas: ");
    scanf("%d", &ht);
    
    if (ht > 50){
        e = ht - 50;
        st = (ht-e)*10 + (e*20);
        printf("O salario total eh %d reais, e o salario excedente eh %d reais",
        st, e*20);
    }else{
        printf("O salario total eh %d reais, e o salario excedente eh 0 reais",
        st);
    };
}